src/test/java/com/artipie/rpm/CliTest.java

```diff
@@ -48,4 +51,57 @@ void testWrongArgumentCount() {
             );
         }
     }
+
+    @Test
+    void testRunWithArgument(@TempDir final Path temp) {
+        try {
+            Cli.main(new String[]{"-nsha256", "-dsha1", "-ftrue", temp.toString()});
+        } catch (final IllegalArgumentException exception) {
+            MatcherAssert.assertThat(
+                String.format("Exception occurred: %s", exception.getMessage()),
+                "Incorrect parameters",
+                new IsEqual<>(exception.getMessage())
+            );
+        }
+    }
+
+    @Test
+    void testRunWithArgumentWithEquals(@TempDir final Path temp) {
+        try {
+            Cli.main(new String[]{"-n=sha256", "-d=sha1", "-f=true", temp.toString()});
+        } catch (final IllegalArgumentException exception) {
+            MatcherAssert.assertThat(
+                String.format("Exception occurred: %s", exception.getMessage()),
+                "Incorrect parameters",
+                new IsEqual<>(exception.getMessage())
+            );
+        }
+    }
+
+    @Test
+    void testRunWithArgumentWithEqualsAndLongopt(@TempDir final Path temp) {
+        try {
+            Cli.main(new String[]{"-nsha256", "-d=sha1", "-filelists=true", temp.toString()});
+        } catch (final IllegalArgumentException exception) {
```

Comment:
@marceloamadeu same here, see my other comment asking whether we should actually be expecting an exception to be thrown in this case.